ebpf-profiling.graphqls

```diff
@@ -41,13 +47,24 @@ type EBPFProfilingTaskCreationResult {
     id: String
 }
 
+# eBPF Profiling task keep alive result
+type EBPFNetworkProfilingKeepResult {
+    # TRUE if the task is keep alive successfully
+    status: Boolean!
+    # error reason when status == FALSE
+    errorReason: String
```

Comment:
1. task does not exist.
2. the target type of task is not `NETWORK`.
3. task already finished when sent keep alive.

Currently, I think when these three cases, the keep-alive would fail.

Reply:
updated.